moped-editor/src/queries/project.js

```diff
@@ -61,7 +61,7 @@ export const SUMMARY_QUERY = gql`
       }
       moped_proj_notes(
         where: { project_note_type: { _eq: 2 }, is_deleted: { _eq: false } }
-        order_by: { date_created: asc }
+        order_by: { date_created: desc }
```

Comment:
i wholeheartedly agree with this change! but i'm slightly confused—were the notes being rendered as timestamp `asc` somewhere? i'm looking in staging and don't see a difference. i guess i'm just curious if this has an affect on the existing  comments tab.